apps/coscrad-frontend/src/store/slices/resources/photographs/selectors/select-loadable-photograph-by-id.ts

```diff
@@ -0,0 +1,51 @@
+import {
+    IDetailQueryResult,
+    IIndexQueryResult,
+    IPhotographViewModel,
+} from '@coscrad/api-interfaces';
+import { createSelector } from '@reduxjs/toolkit';
+import { RootState } from '../../../..';
+import { ILoadable } from '../../../interfaces/loadable.interface';
+import { IMaybeLoadable, NOT_FOUND } from '../../../interfaces/maybe-loadable.interface';
+import { selectLoadablePhotographs } from './select-loadable-photographs';
+
+/**
+ * We would like to use this selector to abstract our 'fetchById' from redux out
```

Comment:
I believe this was worked out on a different branch using a generic hook. Let's remove this file if we won't be using this approach going forward.

Reply:
removed